openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -14,28 +14,28 @@
 from openedx.core.djangoapps.user_api.models import UserPreference
 from .. import PRIVATE_VISIBILITY, ALL_USERS_VISIBILITY
 
-TEST_PASSWORD = "test"
-
-
+@ddt.ddt
```

Comment:
Nit: we can remove @ddt.ddt here now because there are no test points in this class.